server/views/components/primary-nav/template.njk

```diff
@@ -0,0 +1,46 @@
+<div class="moj-primary-navigation">
+
+    <div class="moj-primary-navigation__container">
+
+        <div class="moj-primary-navigation__nav">
+
+            <nav class="moj-primary-navigation" aria-label="Primary navigation">
+
+                <ul class="moj-primary-navigation__list">
+
+                    <li class="moj-primary-navigation__item">
+                        <a class="moj-primary-navigation__link" aria-current="page"
+                           href="/"> {% if params.data.allocations %}
+                                <span id="notifications" style="display: inline-block;"
+                                      class="moj-notification-badge">{{ params.data.allocations | length }}</span>
+                                     {% endif %}
+                            Allocations</a>
+                    </li>
+
+                    <li class="moj-primary-navigation__item">
+                        <a class="moj-primary-navigation__link" href="/">Offender Management</a>
```

Comment:
I think we'll need to get the correct URLs in here. This is basically the top navigation in WMT so all the links should point there. At some point the "Allocations" link will appear in WMT too
 
https://wmt.service.justice.gov.uk/

Reply:
https://github.com/ministryofjustice/wmt-web/blob/main/app/views/includes/header.njk for the paths